workload/scripts/workload-install.ps1

```diff
@@ -146,9 +146,9 @@ if (Get-Command $DotnetCommand -ErrorAction SilentlyContinue)
     $DotnetVersion = Invoke-Expression "& '$DotnetCommand' --version"
     $VersionSplitSymbol = '.'
     $SplitVersion = $DotnetVersion.Split($VersionSplitSymbol);
-    if ($SplitVersion[0] -ne $SupportedDotnetVersion)
+    if ($SplitVersion[0] -lt $SupportedDotnetVersion)
     {
-        Write-Host "Current .NET version is $DotnetVersion. .NET 6.0 SDK is required."
+        Write-Host "Current .NET version is $DotnetVersion. .NET SDK version 6.0 or later is required."
```

Comment:
Is it ok not to check minor version? If you want to check the minor version, 

```powershell
$SupportedDotnetVersion = [Version]"6.0"
....
$CurrentDotnetVersion = [Version]"$($SplitVersion[0]).$($SplitVersion[1])"
if ($CurrentDotnetVersion -lt $SupportedDotnetVersion)
{
    Write-Host "Current .NET version is $DotnetVersion. .NET SDK version $SupportedDotnetVersion or later is required."
}
```

Reply:
fixed